frontend/src/templates/Field/Table/AddRowFooter.tsx

```diff
@@ -51,15 +50,16 @@ export const AddRowFooter = ({
         type="button"
         onClick={handleAddRow}
       >
-        Add another row
+        {t('features.publicForm.components.table.addAnotherRow')}
         <VisuallyHidden>
-          to the table field. {maxRowAriaDescription}
+          {t('features.publicForm.components.table.addAnotherRowAria')}
```

Comment:
Aria for maxRow seemed to be missing here 🫣

Reply:
Yea, deliberate decision on my part. I can readd it, though I just think that it would be a repetition of information. 

wdyt?

Reply:
Understood. Agreed on repetition. No further issues!
=